src/NuGet.Clients/NuGet.CommandLine/Commands/ListCommand.cs

```diff
@@ -137,73 +90,55 @@ private IPackageRepository GetRepository(IEnumerable<string> listEndpoints)
             return partitioned[key: true].ToList();
         }
 
-        public override async Task ExecuteCommandAsync()
+        private IList<Configuration.PackageSource> GetEndpointsAsync()
         {
-            if (Verbose)
+            var configurationSources = SourceProvider.LoadPackageSources()
```

Comment:
I think it's better to keep it in the command as it just consolidates the sources.
I don't think the runner should know about configurations.